raiden_contracts/tests/fixtures/token.py

```diff
@@ -39,6 +39,6 @@ def f(initial_amount: int, decimals: int, token_name: str, token_symbol: str):
 
 
 @pytest.fixture
-def standard_token_contract(deploy_token_contract):
+def standard_token_contract(custom_token):
```

Comment:
Is this actually needed then? We already have `custom_token` and `deploy_token_contract` (for `HumanStandardToken`). This is now just a duplicate for the `custom_token` with a misleading description.

Reply:
MS and Pathfinder still use this fixture, we can remove it later once their code is updated.